packages/admin/src/pages/dashboard/index.test.tsx

```diff
@@ -45,249 +37,201 @@ const mockUserDetails: UserDetails = {
 
-      expect(result).toStrictEqual({
+      expectObjectEquals(result, {
         props: {
           schemes: mockSchemeList,
           userDetails: mockUserDetails,
           oneLoginTransferErrorEnabled: true,
+          migrationSucceeded: null,
         },
       });
     });
+  });
 
-    it('Should return oneLoginTransferErrorEnabled = false', async () => {
-      mockedGetUserSchemes.mockResolvedValue(mockSchemeList);
-      mockedGetLoggedInUsersDetails.mockResolvedValue(mockUserDetails);
-
-      process.env.SESSION_COOKIE_NAME = 'gap-test';
-
-      const oneLoginTransferErrorEnabled =
-        process.env.ONE_LOGIN_MIGRATION_JOURNEY_ENABLED === 'true';
+  describe('Dashboard page render', () => {
+    function getDefaultProps(): InferProps<typeof getServerSideProps> {
+      return {
+        schemes: mockSchemeList,
+        oneLoginTransferErrorEnabled: false,
+        userDetails: mockUserDetails,
+        migrationSucceeded: 'false',
+      };
+    }
 
-      const result = await getServerSideProps({
-        req: { cookies: { 'gap-test': 'testSessionId' } },
-      } as any);
+    it('Should render a page title', () => {
+      render(<Dashboard {...getProps(getDefaultProps)} />);
 
-      expect(result).toStrictEqual({
-        props: {
-          schemes: mockSchemeList,
-          userDetails: mockUserDetails,
-          oneLoginTransferErrorEnabled: oneLoginTransferErrorEnabled,
-        },
-      });
+      screen.getByRole('heading', { name: 'Manage a grant' });
     });
-  });
 
-  describe('Dashboard page render', () => {
     it('Should render the error banner', () => {
       render(
         <Dashboard
-          schemes={mockSchemeList}
-          userDetails={mockUserDetails}
-          oneLoginTransferErrorEnabled={true}
+          {...getProps(getDefaultProps, {
+            oneLoginTransferErrorEnabled: true,
+          })}
         />
       );
-      screen.getByRole('heading', { name: 'PLACEHOLDER FOR ERROR BANNER' });
+
+      screen.getByRole('heading', { level: 2, name: 'Important' });
+      screen.getByText('Something went wrong while transferring your data.');
+      expect(
+        screen.getByRole('link', {
+          name: 'findagrant@cabinetoffice.gov.uk',
+        })
+      ).toHaveAttribute('href', 'mailto:findagrant@cabinetoffice.gov.uk');
     });
 
     it('Should not render the error banner', () => {
-      render(
-        <Dashboard
-          schemes={mockSchemeList}
-          userDetails={mockUserDetails}
-          oneLoginTransferErrorEnabled={false}
-        />
-      );
-      expect(screen.queryByText('PLACEHOLDER FOR ERROR BANNER')).toBeFalsy();
+      render(<Dashboard {...getProps(getDefaultProps)} />);
+
+      expect(
+        screen.queryByRole('heading', { level: 2, name: 'Important' })
+      ).toBeFalsy();
     });
 
-    it('Should render a page title', () => {
+    it('Should render the success banner', () => {
```

Comment:
include the condition producing the behaviour under test in the title

```suggestion
    it('Should render the success banner when migrationSucceeded is true', () => {
```